compiler/rustc_infer/src/infer/canonical/query_response.rs

```diff
@@ -467,11 +467,11 @@ impl<'tcx> InferCtxt<'tcx> {
                     }
                 }
                 GenericArgKind::Const(result_value) => {
-                    if let ty::ConstKind::Bound(debrujin, b) = result_value.kind() {
+                    if let ty::ConstKind::Bound(debruijn, b) = result_value.kind() {
```

Comment:
Both spellings were used, this appears to both be dominant and to match the wider world: https://en.wikipedia.org/wiki/De_Bruijn_sequence
(It's also a thing I got to learn a little about by virtue of reading this repository's code...)